packages/binding.core/src/event.ts

```diff
@@ -42,8 +46,12 @@ export const eventHandler = {
             handlerReturnValue = handler.apply(possiblyUpdatedViewModel, argsForHandler)
           }
         } finally {
-          if (handlerReturnValue !== true) {
-            // Normally we want to prevent default action. Developer can override this be explicitly returning true.
+          // preventDefault in the descriptor takes precedent over the handler return value
+          if (preventDefault !== undefined) {
+            if (unwrap(preventDefault)) { event.preventDefault() }
```

Comment:
Are we still supporting IE <9? I skipped the `event.returnValue` since it shouldn't be necessary for any newer browsers.

Reply:
Even IE11 shouldn't worth any extra work. This browser is a problem to everyone.

Reply:
Anything IE we can / should drop. TKO 4.1 won't guarantee anything IE, because it paralyzes growth. I think the baseline is going to be chrome 50+ (because of Unicode regular expressions), subject to others putting filling in the gaps (which will be possible on a modular basis I hope).

Reply:
Cool. I'll go ahead and remove the `returnValue` code from the existing path, too.